script.js

```diff
@@ -0,0 +1,301 @@
+// quizArrayOfObjects (global)
+const quizArrayOfObjects = []
+
+
+// generate quiz btn true/false 
+const GenerateQuizbtn = () => {
```

Comment:
```suggestion
const generateQuizBtn = () => {
```
Keep your function names in camelCase. It will help you when you will look into React.

Reply:
generateQuizbtn() done